framework/PageInputs/PageSingleSelect.tsx

```diff
@@ -35,27 +45,39 @@ export function PageSingleSelect(props: {
       ref={toggleRef}
       onClick={() => setIsOpen((open) => !open)}
       isExpanded={isOpen}
-      style={{ width: 200 }}
+      style={{ width: '100%', minWidth: 100 }}
```

Comment:
This one is truncating content in certain situations. For instance, 

<img width="671" alt="image" src="https://github.com/ansible/ansible-ui/assets/9053044/edd025ed-bc84-45cc-aa8d-5d2d81dfc32b">